airflow-core/tests/unit/always/test_project_structure.py

```diff
@@ -61,6 +61,7 @@ def test_providers_modules_should_have_tests(self):
         # We should make sure that one goes to 0
         # TODO(potiuk) - check if that test actually tests something
         OVERLOOKED_TESTS = [
+            "providers/amazon/tests/unit/amazon/aws/auth_manager/datamodels/test_login.py",
```

Comment:
This list should go down not up
If there was a reason for a specific file to be added here lets please leave a comment in the code why. Otherwise it will be difficult to review this list later on

Reply:
I agree we should not add items in this list but in my opinion some files do not require tests. Example: https://github.com/apache/airflow/blob/main/providers/amazon/src/airflow/providers/amazon/aws/auth_manager/datamodels/login.py. Creating tests for such classes has no value to me